code/game/objects/machinery/sentries.dm

```diff
@@ -410,7 +411,7 @@
 		return
 
 	var/mob/living/carbon/human/user = usr
-	if(!istype(user))
+	if(!istype(user) && !issilicon(user))
```

Comment:
ew, no.

First you check what type is usr, then you cast.

And do you even need to cast anyway? when are you using user as a human or even as a mob?

Reply:
This stuff will get overwritten by the turret rework in like, two weeks, i don't think it needs to be as sturdy as regular code.